static/static_data.json

```diff
@@ -12,16 +12,16 @@
         "totalNewUndelivered": 238
     },
     "Green": {
-        "totalOldActive": 175,
-        "totalOldInactive": 22,
+        "totalOldActive": 178,
+        "totalOldInactive": 29,
```

Comment:
How are this and the number above both higher?

Reply:
Don't know, I can double check the numbers, but that was the counts on the roster when I checked

Reply:
If that's what the roster says I'm fine to change it, wonder where the discrepancy comes from though